app/src/main/java/com/github/wnder/HistoryActivity.java

```diff
@@ -24,13 +48,73 @@ protected void onCreate(Bundle savedInstanceState) {
         //Set layout
         setContentView(R.layout.activity_history);
 
+        //Buttons to cycle between the images
+        left_button = findViewById(R.id.left_history);
+        right_button = findViewById(R.id.right_history);
+
         //set placeholder
         image = findViewById(R.id.historyImage);
-        image.setImageURI(Uri.parse("android.resource://com.github.wnder/" + R.raw.ladiag));
+        //image.setImageURI(Uri.parse("android.resource://com.github.wnder/" + R.raw.ladiag));
         //When clicked open activity for specific picture history
         image.setOnClickListener((view) -> {
             Intent intent = new Intent(this, PictureHistoryActivity.class);
             startActivity(intent);
         });
+
+        getUserPictures();
+    }
+
+    private void setupButtons() {
+        if(pictureList == null){
+            throw new IllegalArgumentException();
+        }
+        if(pictureList.isEmpty()){
+            //TODO: add default image or just nothing
```

Comment:
Maybe add a message that tells the user that they didn't guess any picture so far